Cogs/StudentCommands.py

```diff
@@ -28,15 +28,12 @@ async def corgme(self, ctx, number=-1):
         # Get images from directory
         images = ['dogs/corgis/' + path.name for path in Path('dogs').rglob('*.*')]
         print(images[0])
-
-        # Pick a random image
-        if number != -1 and (0 < number < len(images)):
-            image = images[number]
-        else:
-            image = images[randint(0, len(images) - 1)]
-
+        #Generates a random number if no number is given
+        if number == -1:
```

Comment:
This changes the functionality of the code. In the original version, this if statement also checks that the specified number is within the range of valid indices.

I'd recommend changing this line to `if 0 <= number < len(images):`

Reply:
Ohh I see, I'll fix that